plugins/LayerView/LayerView.qml

```diff
@@ -315,6 +294,22 @@ Item
                 }
                 text: catalog.i18nc("@label", "Show Infill")
             }
+            CheckBox {
+                checked: view_settings.only_show_top_layers
+                onClicked: {
+                    UM.Preferences.setValue("view/only_show_top_layers", checked ? 1.0 : 0.0);
+                }
+                text: catalog.i18nc("@label", "Only Show Top Layers")
+                visible: UM.LayerView.compatibilityMode
+            }
+            CheckBox {
```

Comment:
Do we even still need this option? It was mostly a workaround for slow top layer generation.